docs/docs/guides/labs/observability-update/asset-health.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Asset health reporting
+description: With asset health criteria, you can quickly identify which datasets are performing well and which need attention.
+sidebar_position: 300
+---
+
+All assets now have a single health status, that combines the status of the most recent materialization, freshness, and asset checks. These statuses appear on the home page, throughout the asset catalog, and in the asset lineage view, and can be used to group and filter your assets.
+
+## Asset health statuses
+
+The overall health status for an asset becomes the most elevated status from among each of the health components.
+
+| Icon                                                                                 | Health status | Latest materialization (unpartitioned) | Latest materialization (partitioned)                                    | Freshness                                  | Asset checks                                                            |
+| ------------------------------------------------------------------------------------ | ------------- | -------------------------------------- | ----------------------------------------------------------------------- | ------------------------------------------ | ----------------------------------------------------------------------- |
+| ![Unknown trend icon](/images/guides/labs/observability-update/status.svg)           | Unknown       | Never materialized                     | All partitions are missing                                              | No freshness policy defined                | No asset checks defined or executed                                     |
+| ![Healthy trend icon](/images/guides/labs/observability-update/successful_trend.svg) | Healthy       | Most recent materialization succeeded  | One or more partitions have executed successfully, and none are failing | Freshness policy is passing                | All asset checks that have executed are passing                         |
+| ![Warning trend icon](/images/guides/labs/observability-update/warning_trend.svg)    | Warning       | Not applicable                         | Not applicable                                                          | Freshness policy is failing with a warning | Some asset checks are failing with a warning                            |
+| ![Degraded trend icon](/images/guides/labs/observability-update/failure_trend.svg)   | Degraded      | Most recent materialization failed     | More than one partition is failing                                      | Freshness policy is failing                | Some asset checks are failing, or had an error on most recent execution |
+
+:::note Current limitations
+
+Health statuses don’t yet take asset observations into account (only materializations).
```

Comment:
If we update this to "Coming soon" instead of "Current limitations", we can also add
- Alerts on health status changes